lib/Parse/Lexer.cpp

```diff
@@ -35,11 +35,9 @@
 // Regex lexing delivered via libSwift.
 static RegexLiteralLexingFn regexLiteralLexingFn = nullptr;
 
-#if SWIFT_SWIFT_PARSER
```

Comment:
It's not, it's just called `SWIFT_BUILD_SWIFT_SYNTAX` now. Was changed in the FetchContent PR. Do I regret that? Maybe a little.

We *could* totally get rid of it now, but I wouldn't for this PR personally.

Reply:
ARGH. Force-pushed now that I realized this, about the same time you told me :)

Reply:
Heh, sorry for not noticing it in here in the first place!